Exercise_3.java

```diff
@@ -19,7 +19,15 @@
     void printMiddle() 
     { 
         //Write your code here
-	//Implement using Fast and slow pointers
+    //Implement using Fast and slow pointers
+    Node slow = head;
+    Node fast = head;
+    while(fast != null && fast.next != null)
+    {
+        slow = slow.next;
+        fast = fast.next.next;
+    }
+    System.out.println("The middle element of the LinkedList is "+ slow.data);

```

Comment:
Please include edge case for a blank list, otherwise at line 30 you will encounter a **NullPointerException**

Reply:
Thank you @samarthjj I will surely work on my mistakes